src/pages/Feed/Feed.js

```diff
@@ -204,13 +210,13 @@ class Feed extends Component {
     })
       .then((res) => res.json())
       .then((fileResData) => {
-        const imageUrl = fileResData.filePath
+        const imageUrl = fileResData.filePath || 'undefined'
 
         // Query create post function
         let graphqlQuery = {
           query: `
-            mutation {
-              createPost(postInput: {title: "${postData.title}", content: "${postData.content}", imageUrl: "${imageUrl}"}) {
+            mutation CreateNewPost($title: String!, $content: String!, $imageUrl: String!) {
```

Comment:
**Careful incorrect GraphQL type**

These variables type must match with GraphQL schema that's defined in our [node-graphql/graphql/schema.js](https://github.com/SuphawitCE/node-graphql/blob/master/graphql/schema.js)

This [line](https://github.com/SuphawitCE/node-graphql/blob/master/graphql/schema.js#L40-L44) 

```graphql
  input PostInputData {
    title: String!
    content: String!
    imageUrl: String!
  }
```